packets/README.md

```diff
@@ -9,7 +8,39 @@
  * 010 = ACK
  * 100 = ?? body length (3,6) (seen from PDM)
 
-#### message_type
+#### Status Packet Exchange 
 
- * 0x03 - get status
- * 0x0a - get status response
+ * PDM (0e01) (Status Request)
+ * PDM (0e01) (Duplicate Packet)
+ * POD (1d18) (Status Response)
```

Comment:
I think the duplicate packets are just when one side doesn't hear the ack, so it retransmits.  Sometimes we see these, and sometimes we don't.  I have SDR captures of status packet exchanges that clearly don't have the duplicates, so I know that we're not just missing them with RFCat.